src/lib/components/particles/relative-time-stamp/index.jsx

```diff
@@ -0,0 +1,19 @@
+import defaultStyles from './style.module.css'
+import { mergeStyles } from '$styles/helpers/mergeStyles'
+import dayjs from 'dayjs'
+import {relativeTime} from "./day-js-relative-time-plugin.js"
+dayjs.extend(relativeTime)
+
+export const RelativeTimeStamp = ({timeStamp, styles}) => {
```

Comment:
Naming... the most difficult of all. I think it would be good to indicate that this is for display to the reader not just a helper functionality. Maybe RelativeTimeSentence ? Will have a think